src/system.h

```diff
@@ -406,6 +406,16 @@ struct System {
 
     done:
 
+        loopv(i, sys->imagelist) sys->imagelist[i]->trefc = 0;
+        doc->rootgrid->ImageRefCount();
+
+        ThreadPool pool(std::thread::hardware_concurrency());   
+        loopv(i, sys->imagelist) {
+            pool.enqueue([](Image *img) {
+                if(img->trefc) img->Display();    

```

Comment:
It be good to put a comment inside `Display` that this runs in multiple threads.. i.e. if anyone ever modifies display, they should not be touching any global resources, and we're assuming that the wx functions its calling are thread safe (do we know?)

Reply:
Ok I added a comment.